src/components/Book.jsx

```diff
@@ -9,14 +9,14 @@ const Book = () => {
   const dispatch = useDispatch();
 
```

Comment:
- [ ] As per project requirements, you should Wrap `App` your with the component from react-redux and not with the RouterProvider . Please check your `App.jsx` file and fix this.
```jsx
  <Provider store={store}>
    <App />
  </Provider>,
```

Reply:
Please check the index.js. You will find that <App />is already wrapped in <Provider store={store}>...</Provider>. [Here](https://github.com/wacoo/react-book-store/blob/630d9877180a88ef3679d9f80dbf23140375d785/src/index.js#L11C5-L13C16) is a link to the code.